pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_command-line-interface.adoc

```diff
@@ -0,0 +1,52 @@
+[id="command-line-interface_{context}"]
+
+= Using the Command Line Interface
+
+Some operations available in the Pantheon user interface can also be performed using commands in the command line interface.
+
+.Using the `Pantheon uploader` via the command line
+
+. Install the uploader script via `setup.sh`:
++
+----
+curl -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh | sh -
+----
++
+. Type `pantheon` in the command line to confirm it was properly installed.
+
+. Navigate to a directory containing test modules.
++
+[options="nowrap", subs="normal"]
+----
+cd _PATH_TO_YOUR_FOLDER_WITH_MODULES_
+----
+. Create the configuration file and edit as needed.
++
+----
+# pantheon --sample > pantheon2.yml
+# vim pantheon2.yml  (or open in your preferred editor)
+----
+. Change the default values for `server` and `repository` as follows:
++
+[options="nowrap",subs="normal"]
+----
+server: link:http://pantheon2.int.us-west.aws.prod.paas.redhat.com[]
+
+repository: _YOUR_KERBEROS_ID_
+----
+. Add the desired file patterns for modules and resources in the file;
+all file patterns are relative to the configuration file location.
++
+* Modules are pre-viewable files, while resources are files which are necessary to build individual modules.
++
+* Any file matching a pattern in the modules section will show up in the search results.
++
+. Run the `Pantheon uploader` command:
++
+[options="nowrap", subs="normal"]
+----
+pantheon push -u _YOUR_USER_ -p _YOUR_PASSWORD_
```

Comment:
This has been a persistent bit of miscommunication.

Up until this line in the document, everything is correct for preparing a git repo to work with Pantheon. However, the only supported way to get documents into an "official Pantheon repository" is to do a git sync via the Pantheon UI. Users would be specifically prevented from a command line invocation as described here.

Having said that, in the future, we'll be supporting user sandboxes that the users can use as a skunkworks area for whatever they like (previewing in-progress docs with CP chroming, sharing those links internally, verifying cross-repo includes when we have them, etc). Docs in a sandbox will not be publishable so we're less strict about how they get there, and therefore CLI upload will be an option for that. However, sandboxes don't exist today, and when they do, we'll likely require an additional 'pantheon push' parameter or another pantheon2.yml entry.

For now, I would recommend keeping the section that involves setting up the yaml, and breaking out the CLI portion into a separate doc to be saved for later when we do have sandbox functionality implemented.

Reply:
I removed the file from this pull request, and also deleted it in https://github.com/redhataccess/pantheon/pull/225
I saved a backup the document itself, but I don't see us wanting to use it in the near future